docs/sphinxext.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+#
+# Gammapy documentation ordering configuration file.
+#
+# GalleryExplicitOrder is a re-implemation of private sphinx_gallery class `_SortKey`.
+#
+
+
+TUTORIAL_SORT_DICT = {
+    #  **Tutorials**
+    #  introductory
+        "overview.py": 0,
+        "analysis_1.py": 1,
+        "analysis_2.py": 2,
+    #  data exploration
+        "hess.py": 0,
+        "cta.py": 1,
+        "fermi_lat.py": 2,
+        "hawc.py": 3,
+    #  data analysis
+    #  1d
+        "cta_sensitivity.py": 0,
+        "spectral_analysis.py": 1,
+        "spectral_analysis_hli.py": 2,
+        "spectral_analysis_rad_max.py": 3,
+        "extended_source_spectral_analysis.py": 4,
+        "ebl.py": 5,
+        "sed_fitting.py": 6,
+        "spectrum_simulation.py": 7,
```

Comment:
spectrum_simulation : 5
sed_fitting : 6
ebl : 7

(The ebl has to come after sed fitting)